mocks/pdl-mock/src/main/java/no/nav/pdl/oversetter/PersonOversetter.java

```diff
@@ -1,58 +1,63 @@
 package no.nav.pdl.oversetter;
 
-import java.time.format.DateTimeFormatter;
-import java.util.List;
-
 import no.nav.foreldrepenger.vtp.testmodell.personopplysning.BrukerModell;
 import no.nav.foreldrepenger.vtp.testmodell.personopplysning.GeografiskTilknytningModell;
 import no.nav.foreldrepenger.vtp.testmodell.personopplysning.PersonModell;
-import no.nav.pdl.Foedsel;
-import no.nav.pdl.Folkeregisterpersonstatus;
-import no.nav.pdl.GeografiskTilknytning;
-import no.nav.pdl.GtType;
-import no.nav.pdl.Kjoenn;
-import no.nav.pdl.KjoennType;
-import no.nav.pdl.Navn;
-import no.nav.pdl.Person;
-import no.nav.pdl.Statsborgerskap;
-
-
-public class PersonAdapter {
+import no.nav.foreldrepenger.vtp.testmodell.personopplysning.StatsborgerskapModell;
+import no.nav.pdl.*;
+
+import java.time.format.DateTimeFormatter;
+
+import static java.util.List.of;
+import static java.util.stream.Collectors.toList;
+
+
+public class PersonOversetter {
     static final DateTimeFormatter DATO_FORMATTERER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
 
-    public static Person oversettPerson(PersonModell personModell) {
+    public static Person oversettPerson(PersonModell personModell, boolean historikk) {
         var person = new Person();
 
         Foedsel fødsel = new Foedsel();
         fødsel.setFoedselsdato(personModell.getFødselsdato().format(DATO_FORMATTERER));
-        person.setFoedsel(List.of(fødsel));
+        person.setFoedsel(of(fødsel));
 
         Navn navn = new Navn();
         navn.setFornavn(personModell.getFornavn().toUpperCase());
         navn.setEtternavn(personModell.getEtternavn().toUpperCase());
         navn.setForkortetNavn(personModell.getEtternavn().toUpperCase() + " " + personModell.getFornavn().toUpperCase());
-        person.setNavn(List.of(navn));
+        person.setNavn(of(navn));
 
-        Statsborgerskap statsborgerskap = new Statsborgerskap();
-        statsborgerskap.setLand(personModell.getStatsborgerskap().getLandkode());
-        person.setStatsborgerskap(List.of(statsborgerskap));
+        person.setStatsborgerskap(
+                historikk
+                        ? personModell.getAlleStatsborgerskap().stream()
+                        .map(PersonOversetter::tilStatsborgerskap)
+                        .collect(toList())
+                        : of(tilStatsborgerskap(personModell.getStatsborgerskap()))
```

Comment:
Her prøver i å lage en mock av PDL og må gjette på hvordan den skal oppføre seg. Det slik at en person kan ha to forskjellige statsborgerskap (https://nav-it.slack.com/archives/C84H68ESC/p1602672636179900?thread_ts=1602672516.179600&cid=C84H68ESC) men det vil ikke denne modellen støtte.